tslint.json

```diff
@@ -5,10 +5,11 @@
     "interface-name": [true, "never-prefix"],
     "max-classes-per-file": [false],
     "member-access": [false, "check-accessor", "check-constructor"],
+    "no-console": [true, ["error", ["warn", "error"]]],
+    "no-unused-variable": [true, { "ignore-pattern": "^_" }],
```

Comment:
Alpha + added the ability to ignore variables via `_`, similar to TypeScript compiler. This combined with new TypeScript setting worked well as TSLint seemed to _not_ check anything but parameters once the TypeScript settings were introduced. 

(See https://github.com/palantir/tslint/pull/1618 for related info).

Reply:
Nice 👍